src/utils.py

```diff
@@ -0,0 +1,76 @@
+import os
+import pandas as pd
+import mysql.connector as mysql
+from dotenv import load_dotenv
+from typing import Optional, Tuple, DataFrame
+
+load_dotenv()
+
+# STEP 1 - connect to server
+def connector(user:str,
+              host:str,
+              db:Optional[str]=None) -> Tuple[str,str]:
+    """
+    Connect via python to mysql server
+
+     Args:
+        user_name (str): person operating
+        host_name (str): usually localhost
+        db_name (Optional[str], optional): Name of database. Defaults to None.
+
+    Returns:
+        Tuple[str,str]: Returns connection, cursor
+
+    """
+    cnx = mysql.connect(user=user,
+                        password=os.getenv('PASSWORD'),
+                        host=host,
+                        db=db)
+    cur = cnx.cursor()
+    return cnx, cur
+
+# STEP 2 - Create db
+def build_db(db:str) -> None:
+    """Build db
+
+    Args:
+        db (str): Input db name
+    """
+    cur.execute(f'DROP DATABASE IF EXISTS {db};')
+    cur.execute(f'CREATE DATABASE {db};')
+    
+# STEP 3 - Read CSV data
+def get_data(file_path:str) -> DataFrame:
+    """_summary_
+
+    Args:
+        file_path (str): _description_
+
+    Returns:
+        DataFrame: _description_
+    """
+    df = pd.read_csv(file_path)
```

Comment:
add this line of code
`df.drop(['Unnamed: 0'], axis=1, inplace=True, errors='ignore')`